tests/Aeon/Calendar/Tests/Unit/Gregorian/DayTest.php

```diff
@@ -217,4 +217,11 @@ public function test_days_between() : void
         $this->assertSame(1, $day1->timeBetween($day2)->inDays());
         $this->assertSame(1, $day2->timeBetween($day1)->inDays());
     }
+
+    public function test_day_static_create() : void
+    {
+        $day = Day::create(2020, 12, 24);
+        $this->assertInstanceOf(Day::class, $day);
```

Comment:
you can remove this assertion, the return type of create method guarantees that it will be an instance of `Day`. Same in MonthTest, besides those two small things, looks good 👍